src/3d/qgscameracontroller.cpp

```diff
@@ -232,9 +232,9 @@ double QgsCameraController::sampleDepthBuffer( const QImage &buffer, int px, int
   // Returns the average of depth values that are not 1 (void area)
   depth = 0;
   int samplesCount = 0;
-  for ( int x = 0; x < mDepthBufferImage.width(); ++x )
+  for ( int x = 0; x < buffer.width(); ++x )
```

Comment:
Where does this change come from? Does this fix an issue?

Reply:
It fixes an issue. In that case, it should be part of a separate commit. This way, this fix could be easily backported.

Reply:
oki doki